drivers/modem/hl7800.c

```diff
@@ -5468,29 +5484,53 @@ int32_t mdm_hl7800_reset(void)
 	return ret;
 }
 
-static int hl7800_power_off(void)
+static void mdm_power_off_work_callback(struct k_work *item)
 {
-	int ret = 0;
+	ARG_UNUSED(item);
+	int ret;
+#if defined(CONFIG_DNS_RESOLVER)
+	struct dns_resolve_context *dnsCtx;
 
-	LOG_INF("Powering off modem");
-	wakeup_hl7800();
-	hl7800_stop_rssi_work();
+	LOG_DBG("Shutdown DNS resolver");
+	dnsCtx = dns_resolve_get_default();
```

Comment:
not sure if this camelCase is intended?

Reply:
Fixed, this was a historical copy paste thing.